bundles/CoreBundle/Migrations/Version20220617145524.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20220617145524 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Drop custom_layouts table in favor of LocationAwareConfigRepository';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('DROP TABLE IF EXISTS custom_layouts');
```

Comment:
As discussed in the daily, this needs to be changed from `drop` to rename table with prefix 'to_be_delete` or so, if it cannot be easily rebuild/restored as it stores extra data.

Reply:
Added warning on downgrade to run command `pimcore:deployment:custom-layouts-rebuild -c` manually as complete data can be restored from def. files.